internal/provider/providers/porkbun/api.go

```diff
@@ -176,12 +181,12 @@ func (p *Provider) updateRecord(ctx context.Context, client *http.Client,
 }
 
 // See https://porkbun.com/api/json/v3/documentation#DNS%20Delete%20Records%20by%20Domain,%20Subdomain%20and%20Type
-func (p *Provider) deleteAliasRecord(ctx context.Context, client *http.Client) (err error) {
+func (p *Provider) deleteRecord(ctx context.Context, client *http.Client, recordType string) (err error) {
```

Comment:
Same as createRecord, the error returned should contain the recordType as wrapping 🙏

Reply:
👍